src/main/index.js

```diff
@@ -68,14 +73,56 @@ if (appConfig.appType === 'development') {
 	}
 }
 
+// NOTE: Has to be set up after user data directory is updated
 const configStore = createConfigStore()
 
+let sentryUser = configStore.get('sentryUser')
+
+// NOTE: The retrieved value may be the default based on config store schema,
+// which is not immediately persisted upon initialization.
+// We want to make sure that this value is persisted upon startup, even if it might be
+// rotated shortly after.
+configStore.set('sentryUser', sentryUser)
+
+if (shouldRotateSentryUser(sentryUser)) {
+	log('Rotating Sentry user')
+	const newSentryUser = generateSentryUser()
+	configStore.set('sentryUser', newSentryUser)
+	sentryUser = newSentryUser
+}
```

Comment:
Doing this in the `createConfigStore` factory would keep everything in one place and reduce this code in the main index.js